apps/extension/client/package.json

```diff
@@ -81,6 +81,20 @@
           "type": "boolean",
           "default": true,
           "description": "Enable nesting the generated typegen files."
+        },
+        "xstate.sendChanges": {
```

Comment:
`sendChanges` seems to be somewhat close to the implementation - from the user perspective I would call it a refresh or a sync (or maybe even something else)

Reply:
Prettier calls them `onSave`, `onPaste` and `onType`.